homeworks/david.tereshchenko_DavidTereshchenko/4-basics/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>JS</title>
+    </head>
+    <body>
```

Comment:
Use HTML for rendering the results. HTML loads faster, allowing users to see some content while the page is loading. Please create the HTML content in the index.html file

Reply:
Get HTML element by data-attributes